### PR TITLE
Feat(serving): add spark mllib model serving

### PR DESCRIPTION
**Pull request by FogDong**
**What this PR does / why we need it**:

Feat(serving): add spark mllib model serving

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @simon-cj @gaocegege @Thrimbda 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: build/serving/mlserver/Dockerfile [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
-    apt-get -y --no-install-recommends install \
-      libgomp1 unzip
+  apt-get -y --no-install-recommends install \
+  libgomp1 unzip
 
 # Use MLServer for serving, see https://github.com/SeldonIO/MLServer
 WORKDIR /workspace
-ADD https://github.com/SeldonIO/MLServer/archive/master.zip .
-RUN unzip master.zip && pip install MLServer-master/[all] && rm -r MLServer-master && rm master.zip
+ADD https://github.com/FogDong/MLServer/archive/master.zip .
```
Comment (FogDong) [inline build/serving/mlserver/Dockerfile:13]: I'll fix it when the mlserver code is ready to be merged in upstream and kelver.

File: scripts/serving/wrapper/preprocessor.py [hunk 2]
```diff
@@ -102,19 +122,30 @@ def start(self):
...
         # Phase 2: Generate 'config.pbtxt' if need
-        if format != 'PMML' and format != 'SKLearn' and format != 'XGBoost':
+        if format != 'PMML' and format not in Preprocessor.mlserver_model:
             self._generate_config_pbtxt(yaml_data)
 
         # Phase 3: Generate 'model setting' if need
-        if format == 'SKLearn' or format == 'XGBoost':
-            self._generate_model_setting(format)
+        if format in Preprocessor.mlserver_model:
+            self._generate_model_setting(format, version)
```
Comment (judgeeeeee) [inline scripts/serving/wrapper/preprocessor.py:149]: Can we rewrite these codes for future scalability ?

Comment (Thrimbda) [reply scripts/serving/wrapper/preprocessor.py:150]: This is awkward... I refactored this in another PR.

**Comment by simon-cj** [comment]
The PR need rebase @FogDong

**Comment by judgeeeeee** [comment]
LGTM

File: build/serving/mlserver/Dockerfile (hunk 1 above)
Comment (gaocegege) [reply build/serving/mlserver/Dockerfile:13]: Should we wait until the change is merged to kleveross?

**Pull request merged**